json-unit-core/src/test/java/net/javacrumbs/jsonunit/core/internal/DifferenceTest.java

```diff
@@ -216,6 +220,13 @@ void shouldSeeExpectedSource() {
         assertThat(listener.getExpectedSource(), equalTo(singletonMap("test", "1")));
     }
 
+    @Test
+    void shouldMatchWithLineSeparatorCustomMatcher() {
```

Comment:
or would be better to update content of test [AllAssertJTest](https://github.com/lukas-krecan/JsonUnit/blob/master/tests/test-base/src/test/java/net/javacrumbs/jsonunit/test/all/AllAssertJTest.java#L36)